web/sites/default/config/webform.webform.apply_tsup.yml

```diff
@@ -186,6 +188,91 @@ elements: |-
           ':input[name="wizard_permit_need"]':
             '!value': existing_permit
       '#prepopulate': true
+    complex_permit_self_doing_work:
+      '#type': radios
+      '#title': 'Will you be the company on site performing this work?'
+      '#required': true
+      '#states':
+        visible:
+          ':input[name="contact_applicant_type"]':
+            value: fun
+          ':input[name="wizard_permit_need"]':
+            '!value': existing_permit
+      '#prepopulate': true
+      '#options': yes_no
+    markup_subcontractor_details:
```

Comment:
Not anything that needs to be fixed, just curious why you're adding conditional logic to each element here and not putting all the self_doing_work elements in a container and putting the conditional logic on the container. You do that with container_fun_parking_warning below. I know there are instances where conditional logic can get confused in nested containers. Just askin.